tests/benchdnn/graph/ref_partition.cpp

```diff
@@ -295,11 +322,14 @@ void ref_partition_t::exec_ops(res_t *res) {
         // A data type to where transform the data will also be provided by the
         // same function since there are corner cases.
         dnnl_data_type_t dt = dnnl_data_type_undef;
-        if ((is_sdpa_pattern || is_gated_mlp_pattern)
+        if ((is_softmax_in_sdpa_pattern || is_matmul_in_sdpa_bwd_pattern
+                    || is_multiply_in_gated_mlp_pattern)
                 && need_unfusable_output_crop(op, dt)) {
             for (size_t i = 0; i < op.out_lts_.size(); i++) {
                 // There's no need to reorder data for undefined or f32 tensors.
                 if (dt == dnnl_data_type_undef || dt == dnnl_f32) continue;
+                // For SoftMax stats, it's f32, no need to reorder.
+                if (op.kind_ == "SoftMax" && i == 1) continue;
```

Comment:
I'd suggest to make it as a part of `need_unfusable_output_crop`. It will probably require to move the function inside the loop and add an index to the signature.